production/ksonnet/loki/config.libsonnet

```diff
@@ -91,13 +93,50 @@
       server: {
         graceful_shutdown_timeout: '5s',
         http_server_idle_timeout: '120s',
-        grpc_server_max_recv_msg_size: 1024 * 1024 * 64,
+        grpc_server_max_recv_msg_size: 100 << 20,
```

Comment:
all these `100 << 20` should probably be kept as a variable somewhere and reused as it seems coordinated in a number of places